README.md

```diff
@@ -1,7 +1,12 @@
 Taskcluster Scopes Utilities
 ============================
 
-Simple utilities to validate scopes, scope-sets, and scope-set satisfiability.
+Simple utilities to validate scopes, scope-sets, and scope-expression satisfiability.
+
+There are two versions of scope utilities available at the current time. First is a new version called [Scope Expressions](#scope-expressions) that allows authorization to
+take place in taskcluster-auth and second is the
+[old version](#local-authorization-old-style) that requires you to verify
+authorization in your service yourself.
```

Comment:
I feel like this proviso is too early -- it belongs below.  Also, I think it's simpler to explain that the old way uses a nested-array structure that is less expressive.  There's no need to bring the authorization service into it.